data/web/inc/init_db.inc.php

```diff
@@ -1482,6 +1486,10 @@ function init_db_schema()
         'msg' => 'db_init_complete'
       );
     }
+
+    // fill quarantine.qhash
+    $pdo->query("UPDATE `quarantine` SET `qhash` = SHA2(CONCAT(`id`, `qid`), 256) WHERE ISNULL(`qhash`)");
```

Comment:
This is also not doing anything, you wanted to retrospectively fill the qhash row right? It does not work.